daphne_service_utils/dapf/src/acceptance/mod.rs

```diff
@@ -878,3 +832,99 @@ pub fn now() -> Now {
         .unwrap()
         .as_secs())
 }
+
+/// An iterator that yields the size of chunks based on a length and a maximum chunk size.
+///
+/// # Example
+/// ```
+/// use dapf::acceptance::Chunker;
+///
+/// assert_eq!(Chunker::of_size(200, 51).collect::<Vec<_>>(), [51, 51, 51, 47]);
+/// ```
+#[derive(Debug, Clone, Copy)]
+pub struct Chunker {
```

Comment:
Where is this used?

Reply:
It's used in this same file to produce an iterator to split the number of report among jobs. Tbh there's probably a shorter way to write this and I'm just suffering from acute skill issue :rofl:

Reply:
There probably is, but never let the perfect be the enemy of the good.

The jank will do just fine.